dev-packages/browser-integration-tests/suites/integrations/httpContext/init.js

```diff
@@ -0,0 +1,20 @@
+import * as Sentry from '@sentry/browser';
+
+window.Sentry = Sentry;
+
+const integrations = Sentry.getDefaultIntegrations({}).filter(
+  defaultIntegration => defaultIntegration.name === 'HttpContext',
+);
```

Comment:
Note: I isolated this similarly to how the SDK was initialized in the original issue so that only this integration is active. 
We don't have to do this but given that all other tests that implicitly check on some of the fields are conventional `Sentry.init` SDK scenarios, I think we can leave it as-is.